fixtures/appliance.py

```diff
@@ -26,15 +27,18 @@ def temp_appliances(count=1, preconfigured=True, lease_time=180):
         preconfigured: True if the appliance should be already configured, False otherwise
         lease_time: Lease time in minutes (3 hours by default)
     """
+    apps = None
     try:
         sprout_client = SproutClient.from_config()
         apps, request_id = sprout_client.provision_appliances(
             count=count, lease_time=lease_time, preconfigured=preconfigured)
         yield apps
+        logger.exception('Error detected during appliance provision')
```

Comment:
Agreed, lcouz, this would make sense in an Exception, but not here, so we can probably just remove this line